src/fmain.pas

```diff
@@ -6688,6 +6688,22 @@ procedure Tfrmain.milistPropertiesClick(Sender: TObject);
   tmpstr:string='';
   paramlist:string='';
   i:integer;
+  itm: TListItem;
+
+  deDestinationText:string='';
+  edtParametersText:string='';
+  edtPasswordText:string='';
+  edtUserText:string='';
+  cbEngineItemText:string='';
+  cbQueueItemText:string='';
+
+  deDestinationTextIsSame:integer=0; // 0 - init; 1 - same; 2 - different
```

Comment:
I'd rather use constants here, instead of 0, 1 and 2 - but I am not sure what is the closest analogue of `C++'es enum` in `Pascal`.

Reply:
You use itm: TListItem; for access to item with short words?, i start this project a long time ago and now i now that you can do this:

```
with frmain.lvMain.Items[i] do
begin
   subitems[columnname]:='';
end;
```

Reply:
You are correct!!! I've completely forgotten about the `with` statement! Indeed, it is much better!